app/controllers/api/v1/partners_controller.rb

```diff
@@ -5,13 +5,15 @@ class Api::V1::PartnersController < ApiController
   def create
     return head :forbidden unless api_key_valid?
 
-    partner = Partner.invite!(email: partner_params[:email],
+    partner = Partner.new(
       diaper_bank_id: partner_params[:diaper_bank_id],
       diaper_partner_id: partner_params[:diaper_partner_id])
+    user = User.invite!(email: partner_params[:email], partner: partner)
 
-    render json: partner.to_json(
-      only: [:id, :email]
-    )
+    render json: {
```

Comment:
We may end up changing this in the future, but for the purposes of this PR, this API should behave exactly like the old one.